frontend/src/components/MeetingRoom/ToolbarOverflowMenu/ToolbarOverflowMenu.tsx

```diff
@@ -0,0 +1,85 @@
+import { ClickAwayListener, PopperChildrenProps } from '@mui/base';
+import { Grow, Paper, Popper } from '@mui/material';
+import { Dispatch, ReactElement, RefObject, SetStateAction } from 'react';
+import ArchivingToggle from '../ArchivingToggle';
+import EmojiGridButton from '../EmojiGridButton';
+import LayoutToggleButton from '../LayoutToggleButton';
+import useSessionContext from '../../../hooks/useSessionContext';
+
+export type ToolbarOverflowMenuProps = {
+  open: boolean;
+  openEmojiGrid: boolean;
+  setOpenEmojiGrid: Dispatch<SetStateAction<boolean>>;
+  anchorRef: RefObject<HTMLButtonElement | null>;
+  handleClickAway: () => void;
+};
+
+/**
+ * ToolbarOverflowMenu Component
```

Comment:
Could we add a short summary on what this component does similarly to ToolbarOverflowButton?